.github/workflows/github-dr-backup.yml

```
@@ -0,0 +1,213 @@
+
+jobs:
+  backup:
+    name: Replicate GitHub source and metadata
+    runs-on: ubuntu-24.04
+    timeout-minutes: 30
+    env:
+      # gh consumes the ephemeral workflow token for read-only API capture.
+      GH_TOKEN: ${{ github.token }}
+      # This URL is a non-secret repository variable and must contain no
+      # username, password, token, or other embedded credential.
+      GITHUB_DR_GITLAB_TARGET: ${{ vars.GITHUB_DR_GITLAB_TARGET }}
```

actionlint (1.7.12)

[error] configuration variable name "github_dr_gitlab_target" must not start with the GITHUB_ prefix (case insensitive). note: see the convention at https://docs.github.com/en/actions/learn-github-actions/variables#naming-conventions-for-configuration-variables

(expression)